optaweb-employee-rostering-backend/src/main/java/org/optaweb/employeerostering/domain/tenant/RosterConstraintConfiguration.java

```diff
@@ -0,0 +1,266 @@
+    @NotNull
+    private DayOfWeek weekStartDay = DayOfWeek.MONDAY;
+
+    // COVID-specific constraints
+    @ConstraintWeight("Low-risk employee assigned to a COVID ward")
+    private HardMediumSoftLongScore lowRiskEmployeeInCovidWardMatchWeight = HardMediumSoftLongScore.ofSoft(1);
+    @ConstraintWeight("Moderate-risk employee assigned to a COVID ward")
+    private HardMediumSoftLongScore moderateRiskEmployeeInCovidWardMatchWeight = HardMediumSoftLongScore.ofSoft(5);
+    @ConstraintWeight("High-risk employee assigned to a COVID ward")
+    private HardMediumSoftLongScore highRiskEmployeeInCovidWardMatchWeight = HardMediumSoftLongScore.ofSoft(10);
+    @ConstraintWeight("Extreme-risk employee assigned to a COVID ward")
+    private HardMediumSoftLongScore extremeRiskEmployeeInCovidWardMatchWeight = HardMediumSoftLongScore.ofHard(1);
+
+    @ConstraintWeight("Required skill for a shift")
+    private HardMediumSoftLongScore requiredSkill = HardMediumSoftLongScore.ofHard(100);
+    @ConstraintWeight("Unavailable time slot for an employee")
+    private HardMediumSoftLongScore unavailableTimeSlot = HardMediumSoftLongScore.ofHard(50);
+    @ConstraintWeight("At most one shift assignment per day per employee")
+    private HardMediumSoftLongScore oneShiftPerDay = HardMediumSoftLongScore.ofHard(10);
+    @ConstraintWeight("No 2 shifts within 10 hours from each other")
+    private HardMediumSoftLongScore noShiftsWithinTenHours = HardMediumSoftLongScore.ofHard(1);
+    @ConstraintWeight("Daily minutes must not exceed contract maximum")
+    private HardMediumSoftLongScore contractMaximumDailyMinutes = HardMediumSoftLongScore.ofHard(1);
+    @ConstraintWeight("Weekly minutes must not exceed contract maximum")
+    private HardMediumSoftLongScore contractMaximumWeeklyMinutes = HardMediumSoftLongScore.ofHard(1);
+    @ConstraintWeight("Monthly minutes must not exceed contract maximum")
+    private HardMediumSoftLongScore contractMaximumMonthlyMinutes = HardMediumSoftLongScore.ofHard(1);
+    @ConstraintWeight("Yearly minutes must not exceed contract maximum")
+    private HardMediumSoftLongScore contractMaximumYearlyMinutes = HardMediumSoftLongScore.ofHard(1);
+
+    @ConstraintWeight("Assign every shift")
+    private HardMediumSoftLongScore assignEveryShift = HardMediumSoftLongScore.ofMedium(1);
+
+    @ConstraintWeight("Undesired time slot for an employee")
+    private HardMediumSoftLongScore undesiredTimeSlot = HardMediumSoftLongScore.ofSoft(1);
+    @ConstraintWeight("Desired time slot for an employee")
+    private HardMediumSoftLongScore desiredTimeSlot = HardMediumSoftLongScore.ofSoft(1);
+    @ConstraintWeight("Employee is not rotation employee")
+    private HardMediumSoftLongScore notRotationEmployee = HardMediumSoftLongScore.ofSoft(1);
+
+    @SuppressWarnings("unused")
+    public RosterConstraintConfiguration() {
+        super(-1);
+    }
+
+    public RosterConstraintConfiguration(Integer tenantId,
```

Comment:
No Covid weights in this constructor? Is this constructor used?

Reply:
I don't think it needs them, as it has the setters and getters. In fact, it's missing getters and setters for many of the non-COVID constraints as well.

Since we don't have UI support for constraint configuration, I do not think we need to worry about this.